src/main/java/bio/terra/cli/businessobject/Workspace.java

```
@@ -71,6 +67,8 @@ private Workspace(WorkspaceDescription wsmObject) {
       this.cloudPlatform = CloudPlatform.GCP;
     } else if (wsmObject.getAzureContext() != null) {
       this.cloudPlatform = CloudPlatform.AZURE;
+    } else {
```

cloudPlatform should never be null, right? Can you throw an exception instead?